adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -252,111 +253,423 @@ a@ Constructs a [code]#depends_on_all_leaves# property instance.
 
 |====
 
-=== Graph
+[[sec:khr-graph-command-graph]]
+== Command-graph class
+
+The [code]#command_graph# class represents a directed acyclic graph of nodes,
+where each node represents a single command with zero or more dependencies
+between them.
+For the list of supported node types see
+<<subsec:khr-graph-supported-node-types>>.
+
+A [code]#command_graph# is associated with a single SYCL device and context on
+which nodes representing device commands will be executed.
+The execution of a graph is considered complete when all of its nodes have
+completed.
+
+A [code]#command_graph# is built up by either recording queue submissions or
+explicitly adding nodes, then once the user is happy that the graph is complete,
```

Comment:
"user" bit doesn't really add extra value, could make this more concise  "then once ~~the user is happy that~~ the graph is complete